# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/url/a-element.html.ini
# Commit: ef6d81f0b298
# Full Hash: ef6d81f0b2989624a9918ec5d790dc2745db20ef
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2018-11-15 09:52:43
# Regressor Bug: 1495313
# File Overlap Count: 1
# Description:
#   Bug 1495313 - Allow password with empty username in URLs r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D11254
# ==============================================================================

diff -r 06b12fd41ff1 -r ef6d81f0b298 testing/web-platform/meta/url/a-element.html.ini
--- a/testing/web-platform/meta/url/a-element.html.ini	Wed Nov 14 19:37:42 2018 +0000
+++ b/testing/web-platform/meta/url/a-element.html.ini	Wed Nov 14 19:10:42 2018 +0000
@@ -17,9 +17,6 @@
   [Parsing: <http::@c:29> against <http://example.org/foo/bar>]
     expected: FAIL
 
-  [Parsing: <http://::@c@d:2> against <http://example.org/foo/bar>]
-    expected: FAIL
-
   [Parsing: <http://foo.com:b@d/> against <http://example.org/foo/bar>]
     expected: FAIL
 
@@ -176,12 +173,6 @@
   [Parsing: <http::b@www.example.com> against <about:blank>]
     expected: FAIL
 
-  [Parsing: <http:/:b@www.example.com> against <about:blank>]
-    expected: FAIL
-
-  [Parsing: <http://:b@www.example.com> against <about:blank>]
-    expected: FAIL
-
   [Parsing: <http:/:@/www.example.com> against <about:blank>]
     expected: FAIL
 
@@ -224,9 +215,6 @@
   [Parsing: <http://@:www.example.com> against <about:blank>]
     expected: FAIL
 
-  [Parsing: <http://:@www.example.com> against <about:blank>]
-    expected: FAIL
-
   [Parsing: <http://example example.com> against <http://other.com/>]
     expected: FAIL
 
@@ -413,9 +401,6 @@
   [Parsing: <tel:1234567890> against <http://example.org/foo/bar>]
     expected: FAIL
 
-  [Parsing: <https://:@test> against <about:blank>]
-    expected: FAIL
-
   [Parsing: <non-special://test:@test/x> against <about:blank>]
     expected: FAIL
 